src/plugins/augeas/augeas.c

```diff
@@ -143,14 +148,18 @@ static int convertToKey (augeas * handle, const char * treePath, void * data)
 	/* fill key values */
 	keySetString (key, value);
 	conversionData->currentOrder++;
-	keySetOrderMeta (key, conversionData->currentOrder);
+	result = keySetOrderMeta (key, conversionData->currentOrder);
+
+	if (result < 0) return result;
```

Comment:
Is the dropping of the key here really what we want? Can you comment (in the source) what you intended here?